src/index.html

```diff
@@ -1,13 +1,53 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="ru" class="product-card">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+
+  <body>
+    <section class="product-card__main" data-qa="card">
+      <div class="product-card__photo-box">
+        <img
+          src="images/imac.jpeg"
+          alt="product-photo"
+          class="product-card__photo-box-photo">
+      </div>
+
+      <div class="product-card__name">
+        <h2 class="product-card__name-title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+        <p class="product-card__name-code">Код товара: 195434</p>
+      </div>
```

Comment:
```suggestion
    <section class="product-card" data-qa="card">
      <div class="product-card__photo-box photo-box">
        <img
          src="images/imac.jpeg"
          alt="product-photo"
          class="photo-box__photo">
      </div>

      <div class="product-card__info info">
        <h2 class="info__title title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
        <p class="info__code code">Код товара: 195434</p>
      </div>
```
классы должны быть примерно такими
product-card__info - задает внешние отступы(марджины) для блока с информацией(margins)
info - задает внутренние стили для блока с информацией(display: flex,...)
info__title - задает внешние отступы для заголовка(margins)
title - отвечает за внутренние стили(font-size, line-height, font-weight, etc)

Reply:
`product-card__photo-box-photo` такие названия классов плохо читаются